custom/opm/reports.py

```diff
@@ -793,18 +793,26 @@ def block(self):
     @property
     def rows(self):
         rows = []
-        for row in self.row_objects + self.extra_row_objects:
+        sorted_objects = self.sort_and_set_serial_numbers(self.row_objects + self.extra_row_objects)
+        for row in sorted_objects:
             rows.append([getattr(row, method) for
-                method, header, visible in self.model.method_map])
+                        method, header, visible in self.model.method_map])
 
-        sorted_rows = sorted(rows, key=lambda item: item[0])
         if self.debug:
             def _debug_item_to_row(debug_val):
                 num_cols = len(self.model.method_map) - 3
                 return [debug_val['case_id'], debug_val['message'], debug_val['traceback']] + [''] * num_cols
 
-            sorted_rows.extend([_debug_item_to_row(dbv) for dbv in self._debug_data])
+            rows.extend([_debug_item_to_row(dbv) for dbv in self._debug_data])
```

Comment:
@czue @esoergel  We could get away with this post processing and extra sorting if we got everything already sorted from ES.